turbinia/workers/__init__.py

```diff
@@ -83,30 +84,29 @@ class TurbiniaTaskResult:
   """Object to store task results to be returned by a TurbiniaTask.
 
   Attributes:
```

Comment:
These are just nitpicks per some of the other comments I left above around capitalization, but seeing task mostly lower cased here and Evidence interchanged between lowercase and not.

Reply:
SGTM